### PR TITLE
Port existing documentation to Jekyll

### PR DESCRIPTION
**Pull request by j-lum**
To simplify the process of writing and publishing documentation, let's migrate from asciidoc to markdown.

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+We are a team based in the [School of Computing, National University of
+Singapore](http://www.comp.nus.edu.sg).
+
+## Project Team
+
+### John Doe
+
+![damithc](images/damithc.jpg)
+
+\[[homepage](http://www.comp.nus.edu.sg/~damithch)\]
```
Comment (damithc) [inline docs/AboutUs.md]: Is the `\` needed? Not usually needed in normal markdown.

File: docs/AboutUs.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+\[[github](https://github.com/damithc)\]
+\[[portfolio](team/johndoe.md)\]
+
+Role: Project Advisor
+
+### John Roe
+
+![lejolly](images/lejolly.jpg)
```
Comment (damithc) [inline docs/AboutUs.md]: Some of these images are too big. use ``?

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -0,0 +1,568 @@
...
+  - does not depend on any of the other three components.
+
+
+<div markdown="span" class="alert alert-info">:information_source: <b>Note: </b>As a more OOP model, we can store a `Tag` list in `Address Book`, which
+`Person` can reference. This would allow `Address Book` to only require
+one `Tag` object per unique `Tag`, instead of each `Person` needing
+their own `Tag` object. An example of how such a model may look like is
+given below.
+
+![BetterModelClassDiagram](images/BetterModelClassDiagram.png)
```
Comment (damithc) [inline docs/DeveloperGuide.md:168]: image
Line break before diagram?

**Review by damithc — CHANGES_REQUESTED** [review]
Added a few comments.

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (j-lum) [reply docs/DeveloperGuide.md:168]: Added a <br> tag because normal markdown breaks don't seem to work.

File: docs/AboutUs.md (hunk 2 above)
Comment (j-lum) [reply docs/AboutUs.md]: Resized everything uniformly.

**Comment by damithc** [comment]
@j-lum Add the preview link to the description. Let's leave the PR like this for now and resume it during the summer.

**Comment by damithc** [comment]
Reminder not to use line wrapping for documentation files. Max line length applies to java files only.  Just write in natural paragraphs.
Reason: When a sentence is updated later, many other lines in the vicinity needs to change to accommodate the change.

**Comment by damithc** [comment]
Superseded by 0be83b242cf13718c419d9996226b147b6cbb437
Sorry, most of your changes got overwritten by my layout tweaks.  I've acknowledged your work in the commit message.

**Pull request closed without merging**